### ISSUE TITLE
windows install failure - must be on a local hard drive?

### ISSUE THREAD

**Issue by xenoguy** [dotnet/cli-lab#176]
I can't install the dotnet-core-uninstall msi (1.2.206301)

it tells me the install needs to be on a local hard drive, but the default folder IS on the local hard drive.

![image](https://user-images.githubusercontent.com/11476694/106530467-60766a80-64ba-11eb-8a03-ce944db0272f.png)

**Comment by sfoslund** [comment]
@joeloff I can't repro locally, is this something you've come across before? Do you have any tips?

**Comment by joeloff** [comment]
@xenoguy Can you launch the installer from the commandline using: ```msiexec /i <path_to_msi> /l*v install.log```

Then when you hit the error, click OK, then cancel and add the install.log to this issue?

@sfoslund the log should hopefully shed some light on why the path validation failed.

**Comment by xenoguy** [comment]
[install.log](https://github.com/dotnet/cli-lab/files/5907747/install.log)

here you go.

**Comment by joeloff** [comment]
Let's see if we can get you unblocked. For now, try running ```msiexec /i <path_to_msi> WIXUI_INSTALLDIR_VALID=1```

Are you running on a VM by any chance or is this a physical machine.

**Comment by xenoguy** [comment]
that worked, thanks!

it's physical hardware.

there's a lot of different versions, could one of them be causing an issue?

![image](https://user-images.githubusercontent.com/11476694/106619128-41b9b780-653e-11eb-93c0-e5961e603182.png)

**Comment by sfoslund** [comment]
Closing as the provided work around fixed the issue.

**Comment by joeloff** [comment]
@xenoguy not sure. I've tracked down a similar issue where one of the DLLs used by the UI dynamically linked to MSVCR100.DLL. The DLL we have should be statically linked to the CRT. I'm going to follow up with a few contacts. Based on the logs you shared, the property used to determine whether that popup you saw is displayed, is never set. I went over the code and that property should always be set to either 0, 1 or an error should be logged if it can't set the property to either value. In your case, it appears that code is never executed, which lead me to believe that its failing to correctly bind to the CRT.
